orchagent/portsorch.cpp

```diff
@@ -773,21 +774,22 @@ bool PortsOrch::addSubPort(Port &port, const string &alias, const bool &adminUp,
         SWSS_LOG_ERROR("%s is not a sub interface", alias.c_str());
         return false;
     }
-    string parentAlias = alias.substr(0, found);
-    string vlanId = alias.substr(found + 1);
```

Comment:
Please retain this logic `if (vlan == "").` In warmboot, the APP_DB entry may not have the vlan field added by intfmgr.